app/src/main/java/com/quran/labs/androidquran/widget/BookmarksWidgetUpdater.kt

```diff
@@ -0,0 +1,26 @@
+package com.quran.labs.androidquran.widget
```

Comment:
These files I moved into a `widget` subpackage. There's already a subpackage called `widgets`, maybe that subpackage should be renamed to `views` instead?

Reply:
`view` singular, yes - but that can be done later outside of this PR